includes/class-newspack-popups-inserter.php

```diff
@@ -589,9 +589,15 @@ public static function popup_shortcode( $atts = array() ) {
 			return;
 		}
 
+		$class_names = '';
+
+		if ( isset( $atts['class'] ) && $atts['class'] ) {
```

Comment:
nit: ` ! empty( $atts['class'] ) ` does the same thing. It checks for isset and checks if the value is not empty. Just be careful that it will consider `0` as empty

Reply:
I usually use `! empty` a lot, not sure why I didn't here. Updated in badb030!